src/opendr/perception/object_detection_2d/nanodet/algorithm/nanodet/model/backbone/ghostnet.py

```diff
@@ -211,7 +211,7 @@ def forward(self, x):
         # 2nd ghost bottleneck
         x = self.ghost2(x)
 
-        x += self.shortcut(residual)
+        x = x + self.shortcut(residual)
```

Comment:
```suggestion
        x += self.shortcut(residual)
```

Reply:
In Onnx with pytorch documentation it has a comment to Avoid in-place operations in some scenarios. I am not sure if it is ok in this particular senario so I change it.